lib/dal/src/migrations/U3001__change_set_pointers.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE change_set_pointers
+(
+    id                    ident primary key        NOT NULL DEFAULT ident_create_v1(),
+    created_at            timestamp with time zone NOT NULL DEFAULT CLOCK_TIMESTAMP(),
+    updated_at            timestamp with time zone NOT NULL DEFAULT CLOCK_TIMESTAMP(),
+    name                  text                     NOT NULL,
+    -- TODO(nick): add once workspaces are added
```

Comment:
Do we want to add the `base_change_set_id` field now?

Reply:
I'm not using it right now, so I do not need it. I changed these migrations because how I accessed them and used them in integration tests also changed. I'll take a look though.

Reply:
Not really a big deal either way, since it'll be a nullable field.